ocaml/xapi-aux/kerberos_encryption_types.ml

```diff
@@ -28,6 +28,21 @@ module Winbind = struct
     | All ->
         "all"
 
+  let to_int = function
+    (*
+     * [X] 0x00000001 DES-CBC-CRC
+     * [X] 0x00000002 DES-CBC-MD5
+     * [X] 0x00000004 RC4-HMAC
+     * [X] 0x00000008 AES128-CTS-HMAC-SHA1-96
+     * [X] 0x00000010 AES256-CTS-HMAC-SHA1-96
+     * *)
+    | Strong ->
```

Comment:
Please use `0x00000001` and so on - it is legal syntax for integers. I believe `to_int` is misleading; the integer represents and encoding that represents the ciphers to use. If possible, the fact that this is an int should be hidden.